SH4RE/SH4RE/ViewListing/ViewListingView.swift

```diff
@@ -0,0 +1,158 @@
+//
+//  ViewListingView.swift
+//  SH4RE
+//
+//  Created by Hannah Brooks on 2022-11-24.
+//
+
+import SwiftUI
+import Firebase
+import FirebaseStorage
+
+//Stuff to know for database
+//ViewListing is passed a listing struct define in ListingViewModel
+//We map the fields from the listing struct to class variable in .OnAppear
+//View listing is passed and image which is set as the classes image variable being displayed
+//if you need to create more variables create filler class variable and we will connect to database in later commit
+
+struct ViewListingView: View {
+    
+    //parameters passed in from search nav link
+    var listing: Listing
+    var image: UIImage
+    
+    let screenSize: CGRect = UIScreen.main.bounds
+    var numberOfStars = 4
+    var hasHalfStar = true
+    var numberOfReviews = 3
+    var numberOfImages = 3 // should become images.length or something
+    @State var description:String = ""
+    @State var title:String = ""
+    @State var price:String = ""
+    
+    var body: some View {
+        
+        ZStack {
+            Color("BackgroundGrey").ignoresSafeArea()
+            VStack(alignment: .leading) {
```

Comment:
Should this be wrapped in a scroller? to prevent the big gap that appears?
<img width="324" alt="Screenshot 2022-12-01 at 5 37 45 PM" src="https://user-images.githubusercontent.com/46950469/205173918-3cb60d81-a510-43ad-a3b8-c7e0079da7b2.png">
<img width="312" alt="Screenshot 2022-12-01 at 5 38 29 PM" src="https://user-images.githubusercontent.com/46950469/205173970-f3644fa4-9f94-4ee3-88a7-acca0d4921c7.png">

Reply:
looking at this now

Reply:
i dont get the gap, but i will make a ticket for this just so that pr can get merged